source/server/active_stream_listener_base.h

```diff
@@ -84,8 +84,13 @@ class ActiveStreamListenerBase : public ActiveListenerImplBase,
 
   void onSocketAccepted(std::unique_ptr<ActiveTcpSocket> active_socket) {
     // Create and run the filters
-    config_->filterChainFactory().createListenerFilterChain(*active_socket);
-    active_socket->continueFilterChain(true);
+    if (config_->filterChainFactory().createListenerFilterChain(*active_socket)) {
+      active_socket->continueFilterChain(true);
+    } else {
+      // If create listener filter chain failed, it means the listener is missing
+      // config due to the ECDS. Then close the connection directly.
+      active_socket->socket_->close();
```

Comment:
thanks! that is good question. If we return here, then we will lose the access log at line 103. I guess it would be good to keep the access log, WDYT?